manual/src/refman/extensions/bindingops.etex

```diff
@@ -92,6 +96,26 @@ let sum3 z1 z2 z3 =
     (fun ((x1, x2), x3) -> x1 + x2 + x3)
 \end{caml_example}
 
+
+\subsection{ss:letops-punning}{Short notation for variable bindings}
+
+(Introduced in 4.13.0)
+
+When the expression being bound is a variable, it can be convenient to
+use the shorthand notation "let+ x in ...", which expands to "let+ x =
+x in ...".  Using this notation, the "sum3" function above can be
+written more concisely as:
```

Comment:
I think that somewhere in the explanation text the name "let punning" should appear (not just in the TeX label), because this is how we talk about the feature (for example in the changelog) and users should be able to locate the feature by its name, not just in github pull requests.

Reply:
I checked the manual under "records" and found that the term "punning" isn't used there (it's called "short notation for records" or something) and so concluded that this name was somehow unofficial.

Reply:
Let's make it official then :-)